aregion.cpp

```diff
@@ -2348,8 +2348,8 @@ ARegion *ARegionList::FindNearestStartingCity(ARegion *start, int *dir)
 int ARegionList::GetPlanarDistance(ARegion *one, ARegion *two,
 		int penalty, int maxdist)
 {
-	if (one->zloc == ARegionArray::LEVEL_NEXUS ||
-			two->zloc == ARegionArray::LEVEL_NEXUS)
+	// make sure you cannot teleport into or from the nexus
+	if (Globals->NEXUS_EXISTS && (one->zloc == ARegionArray::LEVEL_NEXUS ||	two->zloc == ARegionArray::LEVEL_NEXUS))
```

Comment:
On a world without a nexus level, this *always* returned 10000 for planar distance :/